pom.xml

```diff
@@ -754,14 +754,6 @@
         </plugins>
       </build>
       <dependencyManagement>
-        <dependencies>
-          <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>31.0.1-jre</version>
-            <scope>compile</scope>
-          </dependency>
-        </dependencies>
```

Comment:
Can we skip this as it is not showing up in the list of guava CVE's
`/opt/connectors/lib/salesforce/libs/io.cdap.plugin.salesforce-plugins-1.6.13.jar/lib/guava-13.0.1.jar`

<img width="1774" height="738" alt="image" src="https://github.com/user-attachments/assets/e9cdac88-bd23-48a9-a029-2f710ea9ad7b" />

Reply:
Also, exclude the transitive dependency fetched through `hydrator-common`:`2.10.0` and explicitly add the upgraded version.
<img width="649" height="295" alt="image" src="https://github.com/user-attachments/assets/b671f201-baa2-4b2d-b272-7de6503fd9c6" />